### PR TITLE
[TD-12173]<docs>: fix page layout error in official doc of python connector

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: documentation20/cn/08.connector/docs.md [hunk 1]
```diff
@@ -557,25 +556,30 @@ Python连接器支持的系统有：Linux 64/Windows x64
...
 
-**Linux**
+#### Linux
 
-用户可以在源代码的src/connector/python（或者tar.gz的/connector/python）文件夹下找到connector安装包。用户可以通过pip命令安装： 
+用户可以在源代码的src/connector/python（或者tar.gz的/connector/python）文件夹下找到connector安装包。用户可以通过pip命令安装：
 
-​		`pip install src/connector/python/`
+```bash
+pip install src/connector/python/
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/docs.md]: we can use 'pip3 install taospy' now.

File: documentation20/cn/08.connector/docs.md [hunk 2]
```diff
@@ -557,25 +556,30 @@ Python连接器支持的系统有：Linux 64/Windows x64
...
+```
 
 或
 
-​		`pip3 install src/connector/python/`
+```bash
+pip3 install src/connector/python/
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/docs.md]: we can use 'pip3 install taospy' now.

File: documentation20/cn/08.connector/docs.md [hunk 3]
```diff
@@ -557,25 +556,30 @@ Python连接器支持的系统有：Linux 64/Windows x64
 
 ### Python连接器安装
 
-**Linux**
+#### Linux
 
-用户可以在源代码的src/connector/python（或者tar.gz的/connector/python）文件夹下找到connector安装包。用户可以通过pip命令安装： 
+用户可以在源代码的src/connector/python（或者tar.gz的/connector/python）文件夹下找到connector安装包。用户可以通过pip命令安装：
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/docs.md]: How about: 用户可以从 PyPI 安装或使用源代码安装... ?

**Comment by zitsen** [comment]
Updated.

**Review by dingbo8128 — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**